1-JavaScript/Week4/homework/ex5-wallet/index.js

```diff
@@ -86,7 +86,7 @@ const quiz = {
       b: 'anonymous, transferInto', 
       c: 'transferInto, anonymous' 
     },
-    answer: undefined,
+    answer: c,
```

Comment:
@Mehran-Banbersta Correct! The answer must be a String ' '
```suggestion
    answer: 'c',
```